lib/stripe/converter.ex

```diff
@@ -166,11 +166,11 @@ defmodule Stripe.Converter do
     defp warn_unknown_object(%{"object" => object_name}) do
       require Logger
 
-      Logger.warn("Unknown object received: #{object_name}")
+      Logger.warning("Unknown object received: #{object_name}")
     end
   end
 
-  if Mix.env() == :prod do
+  if Mix.env() == :dev do
```

Comment:
Was this a bug? Without the full context here, I think it makes sense for us to not allow "extra keys" if we're in Prod.

The flexible behaviour seems like a nice-to-have for DevEx only?

Reply:
I was messing around with it and it ended up with `:dev`, but this should be `:prod`.